tests/neptune/test_experiment.py

```diff
@@ -150,6 +150,40 @@ def build_call(tags_list):
             build_call(['tag1', 'tag2', 'tag3'])
         ])
 
+    def test_delete_artifact(self):
+        # given
+        backend = mock.MagicMock()
+        project = a_project()
+        experiment = Experiment(
+            backend,
+            project,
+            an_experiment_id(),
+            a_uuid_string()
+        )
+
+        # and
+        def build_call(paths):
+            return call(
+                experiment=experiment,
+                paths=paths
+            )
+
+        # when
+        experiment.delete_artifacts('/an_abs_path_in_exp_output')
+        experiment.delete_artifacts('/../an_abs_path_in_exp')
+        experiment.delete_artifacts('/../../an_abs_path_in_prj')
```

Comment:
Should fail.

Reply:
actually no, it shouldn't . If follows the behaviour of linux standard commands.
execute following script:
```
touch /tmp/test
rm /../tmp/test
```
It will successfully remove the file

Reply:
Right.